mrmustard/training/parametrized.py

```diff
@@ -77,7 +80,7 @@ def _traverse_parametrized(object_: Any, extract_type: Parameter) -> Generator:
     """
 
     for obj in object_:
-        if isinstance(obj, Sequence):  # pylint: disable=isinstance-second-argument-not-valid-type
+        if isinstance(obj, List):  # pylint: disable=isinstance-second-argument-not-valid-type
```

Comment:
Why modifying this? A ``List`` is a ``Sequence``. Also, this prevents the parametrized class from extracting parameters inside a tuple, for example.

Reply:
A string is a `Sequence` too though: this was creating problems with the recursive crawling